src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -189,7 +189,7 @@ const getAvailablePaymentMethods = (
       value={paymentMethod}
       label={
         <>
-          <CreditCardIcon type="Unknown" fill="black60" />
+          <CreditCardIcon type="Unknown" fill="black100" />
```

Comment:
none of the credit card icon types or fill colours gives it a darker look – I suggest we update this icon in Palette to provide a darker look